src/ecrc-api/src/main/resources/application.properties

```diff
@@ -16,6 +16,12 @@ info.app.name=@project.name@
 info.app.description=@project.description@
 info.app.version=@project.version@
 
+#JWT Configs
+ecrc.jwt-header=${ECRC_JWT_HEADER}
```

Comment:
Would these require an associated change to the properties file?

Reply:
Yes they are setup in the properties config.